Diary/Domain.Entities/Domain/Administration/HabitDiaryOwner.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Diary.Core.Domain.BaseTypes;
+using Diary.Core.Domain.Diary;
+
+namespace Diary.Core.Domain.Administration
+{
+    public class HabitDiaryOwner : BaseEntity
+    {
+        public required string Name { get; set; }
+        public required string Email { get; set; }
```

Comment:
How will the Email property be used in this microservice? Should he be removed from here?

Reply:
As I understand it, email is present in the authentication service, maybe it is only needed there?

Reply:
hm, may be. Ok, i will remove